mlir/lib/Conversion/AIRToAIEPass.cpp

```diff
@@ -899,11 +899,14 @@ void allocL1Buffers(ModuleOp m,
   (void)applyPatternsAndFoldGreedily(m, std::move(patterns));
 }
 
-AIE::ObjectFifoCreateOp
-createObjectFifo(OpBuilder &builder, AIE::AIEObjectFifoType datatype,
-                 Value prodTile, std::vector<Value> consTile, int depth) {
+AIE::ObjectFifoCreateOp createObjectFifo(OpBuilder &builder,
+                                         AIE::AIEObjectFifoType datatype,
+                                         Value prodTile,
+                                         std::vector<Value> consTile, int depth,
```

Comment:
```suggestion
                                         cont std::vector<Value> &consTile, int depth,
```
might be enough?

Reply:
fixed in https://github.com/Xilinx/mlir-air/pull/231